src/main/java/com/iexec/worker/tee/pre/Main.java

```diff
@@ -18,7 +18,16 @@
 
 public class Main {
 
+    static PreComputeAppRunner preComputeAppRunner;
+
     public static void main(String[] args) {
-        PreComputeAppRunner.start();
+        final int exitCode = getPreComputeAppRunner().start();
+        System.exit(exitCode);
+    }
+
+    static PreComputeAppRunner getPreComputeAppRunner() {
```

Comment:
I'm a bit puzzled by this method.
We are in an application which needs to be slick and fast.
Here, it looks like a method added to allow more testing, but without real value during application runtime.

I would prefer to keep
`final int exitCode = new PreComputeAppRunner().start()`
even at the cost of a slightly worse code coverage result

Reply:
Weird thing is `catchSystemExit()` doesn't show proper coverage of `System.exit(exitCode)` in Sonar.